leejihee/third/cards/index.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/6212e88fd3.js" crossorigin="anonymous"></script>
+</head>
+<body>
+    <article>
+        <div class="item financial-app">
+            <div class="front">
+                <h1>Financial APP</h1>
+                <p>25 Apr 2030</p>
+            </div>
+            <div class="back">
+                <button onclick="alert('Financial APP')">Click</button>
+                <i class="fa-solid fa-computer-mouse"></i>
+            </div>
+        </div>
+        <div class="item graphic-design">
+            <div class="front">
+                <h1>Graphic Design</h1>
+                <p>18 May 2030</p>
+            </div>
+            <div class="back">
+                <button onclick="alert('Graphic Design')">Click</button>
+                <i class="fa-solid fa-computer-mouse"></i>
+            </div>
+        </div>
+        <div class="item artifact-model">
+            <div class="front">
+                <h1>Artifact Model</h1>
+                <p>17 Nov 2030</p>
+            </div>
+            <div class="back">
+                <button onclick="alert('Artifact Model')">Click</button>
+                <i class="fa-solid fa-computer-mouse"></i>
+            </div>
+        </div>
+        <div class="item operating-system">
+            <div class="front">
+                <h1>Operationg System</h1>
```

Comment:
```suggestion
                <h1>Operating System</h1>
```
오타가 있었네요! 항상 오타 조심해야 합니다.
비주얼스튜디오코드에 code spell checker 라는 확장프로그램이 있는데 이거 설치하면 오타가 있는 부분을 체크해줘서 놓치지 않도록 해줍니다! 이거 설치해봅시다😁
![image](https://user-images.githubusercontent.com/87893624/182084866-259597bc-cabe-49a1-81af-46c36cc616f3.png)
![image](https://user-images.githubusercontent.com/87893624/182084936-7453bf9f-a7d3-4596-9472-de3410978fac.png)